index.html

```diff
@@ -736,35 +742,69 @@
                                 var inputPort, outputPort, controlPort, commandPort;
                                 if (cellvar.model.in.height == null) {
                                     inputPort = 0;
-								}
+                                }
                                 else {
                                     inputPort = cellvar.model.in.height;
-								}
+                                }
                                 if (cellvar.model.out.height == null) {
                                     outputPort = 0;
                                 }
-								else {
+                                else {
                                     outputPort = cellvar.model.out.height;
-								}
+                                }
                                 if (cellvar.model.evtin.height == null) {
                                     controlPort = 0;
-								}
+                                }
                                 else {
                                     controlPort = cellvar.model.evtin.height;
-								}
+                                }
                                 if (cellvar.model.evtout.height == null) {
                                     commandPort = 0;
-								}
+                                }
                                 else {
                                     commandPort = cellvar.model.evtout.height;
-								}
+                                }
+                                var flip = 'false', mirror = 'false';
+                                function Styling(style)
```

Comment:
a) These cannot be global variables
b) It should be a boolean and not a string!